core/src/main/java/org/kohsuke/stapler/lang/Klass.java

```diff
@@ -56,7 +57,13 @@ public Class toJavaClass() {
     }
 
     public List<FieldRef> getDeclaredFields() {
-        return navigator.getDeclaredFields(clazz);
+        try {
+            return navigator.getDeclaredFields(clazz);
+        } catch (AbstractMethodError err) {
```

Comment:
Actually this handling supresses error I've expected to catch in tests, but tests *on aligned stapler module versions* pass independently of this hack